README.adoc

```diff
@@ -122,6 +122,14 @@ To build the application, run the Maven `package` goal from the command line in
 mvn clean package
 ```
 
+Run the following commands to build and containerize the application:
+
+[role='command']
+```
+docker build -t system:1.0-SNAPSHOT system/.
```

Comment:
There're 4 services, system, inventory, gateway and job, plus 2 other services, kafka and zookeeper. I guess all need to be built with the new `docker build` command? @proubatsis what do you think?

Reply:
Yes, they all need to be built.